test/unit/composite.js

```diff
@@ -333,25 +351,13 @@ describe('composite', () => {
     it('invalid left', () => {
       assert.throws(() => {
         sharp().composite([{ input: 'test', left: 0.5 }]);
-      }, /Expected positive integer for left but received 0.5 of type number/);
+      }, /Expected integer for left but received 0.5 of type number/);
     });
 
     it('invalid top', () => {
       assert.throws(() => {
-        sharp().composite([{ input: 'test', top: -1 }]);
-      }, /Expected positive integer for top but received -1 of type number/);
-    });
-
-    it('left but no top', () => {
```

Comment:
The "left but no top" and "top but no left" tests have been removed; is this intentional?

Reply:
Added the test cases back :)